test/functional/p2p-fullblocktest.py

```diff
@@ -94,228 +96,123 @@ def set_test_params(self):
         self.tip = None
         self.blocks = {}
 
-    def add_options(self, parser):
-        super().add_options(parser)
-        parser.add_option("--runbarelyexpensive", dest="runbarelyexpensive", default=True)
```

Comment:
In commit "Tidy up p2pfullblocktest"

Can you note that this option was removed in the commit message (and maybe say why it wasn't useful?)

Reply:
Yep. Added to commit message.